src/app/pages/market/ProductAction.php

```diff
@@ -59,21 +63,34 @@ public function __invoke($request, $response, $args)
           $docArtifacts[] = $artifact;
         }
       }
+
+      $mavenArtifacts = array_filter($mavenArtifacts, fn(MavenArtifact $a) => !$a->hide());
     }
 
     $installButton = self::createInstallButton($request, $product, $version);
     
     $getInTouchLink = 'https://www.axonivy.com/marketplace/contact/?market_solutions=' . $product->getKey();
 
+    if (!empty($version)) {
```

Comment:
why do you not check if the version is empty in the ProductMavenArtifactDownloader, like it's done in the other places (OpenAPIProvicer and ProductDescription)?

Reply:
will make it beautifuller later. some artifacts has no version at all. e.g. employee-on-boarding